frontend/src/app/testQueue/QueueItem.tsx

```diff
@@ -821,7 +817,11 @@ const QueueItem = ({
             step="60"
             value={moment(dateTested).format("HH:mm")}
             onChange={(e) => handleTimeChange(e.target.value)}
-            disabled={deviceTypeIsInvalid() || specimenTypeIsInvalid()}
+            disabled={
+              deviceTypeIsInvalid() ||
+              specimenTypeIsInvalid() ||
+              saveState === "editing"
```

Comment:
Just to make sure [we're not reproing the bug that motivated this change in the first place](https://github.com/CDCgov/prime-simplereport/pull/2125), `saveState === "editing"` takes the place of the debounce effect here, correct? I can't come up with a corner case where they're not equivalent, but flagging in case it raises something for you too.

Reply:
Thanks for bringing this up! I notice now that the date picker is pretty much broken with this change. Let me revise this to fix that.

Reply:
Just wanted to mention that @mpbrown is working on this ticket #6079 that will basically replace the current implementation of the testcard.

Reply:
Ok, since that's the case. I'll just add a wait in the test for the time being.

Reply:
I'll make sure we tag you on the new testcard ticket to ensure we have this case covered in the new ticket